stories/components/SQFormStoryWrapper.tsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import CheckMarkIcon from '@material-ui/icons/CheckCircle';
-import Grid from '@material-ui/core/Grid';
+import Grid, {GridProps} from '@material-ui/core/Grid';
 import {SQForm, SQFormIconButton} from '../../src';
 import {
   Snackbar,
   SnackbarProvider,
-  useSnackbar
+  useSnackbar,
 } from 'scplus-shared-components';
+import {FormikValues} from 'formik';
+import {AnySchema} from 'yup';
+
+interface SQFormStoryWrapperProps {
```

Comment:
I think only `children` and `initialValues` should be required

Reply:
Good call. Thanks.